src/components/Button/Button.tsx

```diff
@@ -5,7 +5,7 @@ import useButtonStyles from './useButtonStyles';
 import Spinner from '../Spinner';
 import Icon, { IconProps } from '../Icon';
 
-export interface ButtonProps extends NativeAttributes<'button'>, Pick<BoxProps, 'as' | 'to'> {
```

Comment:
remove as we don't want to be using buttons for links. We'll wrap links around them, while doing `as="span"` on the  `Button`